docs/run/primitives-get-started.mdx

```diff
@@ -79,28 +54,24 @@ print(f">>> Observable: {observable.paulis}")
     The circuit and observable need to be transformed to only use instructions supported by the system (referred to as *instruction set architecture (ISA)* circuits). We'll use the transpiler to do this.
 
 ```python
-pm = generate_preset_pass_manager(backend=backend)
+pm = generate_preset_pass_manager(optimization_level=1, backend=backend)
 isa_circuit = pm.run(circuit)
 isa_observable = observable.apply_layout(isa_circuit.layout)
 ```
-  </TabItem>
-</Tabs>
 
 ### 3. Initialize Qiskit Runtime Estimator
 
-We want to use the Qiskit Runtime `Estimator`, so we initialize an instance of `qiskit_ibm_runtime.Estimator` instead of `qiskit.primitives.Estimator`. 
-
-When you initialize the `Estimator`, you must pass in the system or simulator you previously selected as the target.  You could also do this within the `session` parameter. 
+When you initialize the Estimator, pass it the system or simulator you previously selected as the target.
 
+To use the V2 Estimator, import the **EstimatorV2** class. The **Estimator** class still refers to the version 1 Estimator, for backwards compatibility.
 
 <Tabs>
   <TabItem value="EstimatorV2" label="V2 Estimator">
-    When using V2 Estimator, for backward compatibility, you must explicitly import EstimatorV2. After V1 support ends, you can use either `import` command (`from qiskit_ibm_runtime import Estimator` or `from qiskit_ibm_runtime import EstimatorV2`) to import EstimatorV2. 
 
 ```python
-from qiskit_ibm_runtime import EstimatorV2 as Estimator
+from qiskit_ibm_runtime import EstimatorV2
 
-estimator = Estimator(backend=backend)
+estimator = EstimatorV2(backend=backend)
```

Comment:
I think we should keep the `import PrimitiveV2 as Primitive` things so that it remains consistent with all the other pages and migration guide

Reply:
Done.